### PR TITLE
Font URLs

### PR DESCRIPTION
**Pull request by ronyeh**
Update URLs for web fonts.
Update glyph and OTF inspectors to view new fonts.

### CODE REVIEW

File: tools/fonts/inspector/glyphs.html [hunk 1]
```diff
@@ -177,70 +134,69 @@ <h1 id="title">Glyph Inspector</h1>
...
       }
 
-
       function codepoint2decimal(codepoint) {
         // Return the decimal value of 'XXXX' of codepoint, which has a string format 'U+XXXX'
         return parseInt(codepoint.substring(2), 16);
       }
 
-      function drawGlyph(canvasId,glyphCode,glyphName,fontName) {
+      function drawGlyph(canvasId, glyphCode, glyphName, fontName) {
```
Comment (ronyeh) [inline tools/fonts/inspector/glyphs.html:142]: I did not edit this function. It was auto-formatted by prettier.

File: tools/fonts/inspector/glyphs.html [hunk 2]
```diff
@@ -257,11 +213,10 @@ <h1 id="title">Glyph Inspector</h1>
           drawGlyph(canvasId, glyphCode, glyphName, fontName);
           var alternateCodepoint = SMuFLGlyphInfo[glyphName].alternateCodepoint;
           if (alternateCodepoint) {
-            glyphCode = alternateCodepoint.substring(2)
+            glyphCode = alternateCodepoint.substring(2);
```
Comment (ronyeh) [inline tools/fonts/inspector/glyphs.html:216]: This was a eslint fix.

File: tools/fonts/inspector/otf.html [hunk 3]
```diff
@@ -123,35 +123,47 @@
...
-        'MuseJazz Text': 'musejazztext/musejazztext.otf',
-        'Petaluma Script': 'petalumascript/petalumascript.otf',
+        Academico: null,
+        Edwin: 'edwin-roman',
+        'Finale Ash Text': null,
+        'Finale Broadway Text': null,
+        'Finale Jazz Text': null,
+        'Finale Maestro Text': 'finalemaestrotext-regular',
+        'MuseJazz Text': null,
+        Nepomuk: 'nepomuk-regular',
```
Comment (ronyeh) [inline tools/fonts/inspector/otf.html:154]: Nepomuk is the text font suggested by the Sebastian designers.

https://github.com/fkretlow/sebastian/blob/master/fonts/Sebastian.json

File: tools/fonts/inspector/otf.html [hunk 4]
```diff
@@ -123,35 +123,47 @@
...
+      // Other OTF files have custom names, so we map them to the font names.
       const textFonts = {
-        Academico: 'academico/academico.otf',
-        'Roboto Slab': 'robotoslab/robotoslab-regular-400.otf',
-        Edwin: 'edwin/edwin-roman.otf',
-        'Finale Maestro Text': 'finalemaestrotext/finalemaestrotext-regular.otf',
-        'MuseJazz Text': 'musejazztext/musejazztext.otf',
-        'Petaluma Script': 'petalumascript/petalumascript.otf',
+        Academico: null,
+        Edwin: 'edwin-roman',
```
Comment (ronyeh) [inline tools/fonts/inspector/otf.html:148]: Edwin is the text font suggested by the Leland designers:

https://github.com/MuseScoreFonts/Leland/blob/main/leland_metadata.json

File: tools/fonts/inspector/otf.html [hunk 5]
```diff
@@ -123,35 +123,47 @@
...
+        'Finale Ash Text': null,
+        'Finale Broadway Text': null,
+        'Finale Jazz Text': null,
+        'Finale Maestro Text': 'finalemaestrotext-regular',
```
Comment (ronyeh) [inline tools/fonts/inspector/otf.html:152]: These are the text fonts suggested by Finale to pair with each Finale font.

The naming convention does not match the Bravura Text, Leland Text, Petaluma Text, etc.... Those fonts are actually music engraving fonts designed for text heavy documents.

**Review by rvilarl — APPROVED** [review]

**Pull request merged**